### DISCUSSION TITLE
Fill_between no longer working?

### DISCUSSION THREAD

**Discussion by bill8chen** [highfestiva/finplot#457] [Q&A] [answered]
Hello everyone
I use the code in [[BUG] No separate "candle_bear_body_color" #381](https://github.com/highfestiva/finplot/issues/381) provided by gabrsafwat, but the band color isn’t shown in finplot 1.9.3, it was shown in finplot 1.9. 0.
Please help. Thanks.

**Comment by highfestiva** [comment]
Strange, couldn't find the issue at a glance, will look into it when I have time.

**Comment by highfestiva** [comment, marked as answer]
This was caused by a bug in pyqtgraph, so it took a while to find. I was however able to circumvent it, fix in 3d573573653b039fb8deab9db4a54615ead3d885.

**Reply by bill8chen** [reply]
This fix does solve my problem.
Thanks for your quick response and hard work.
